.github/workflows/feg-workflow.yml

```diff
@@ -102,12 +102,24 @@ jobs:
         with:
           name: Unit Test Results
           path: "/tmp/test-results"
+      - name: Lint radius src
+        if: always()
+        id: radius-src-lint
+        working-directory: feg/radius/src
+        run: |
+          ./run.sh lint
       - name: Run radius src unit tests
         if: always()
         id: radius-src-unit-tests
         working-directory: feg/radius/src
         run: |
           ./run.sh test
+      - name: Lint radius lib
+        if: always()
+        id: radius-lib-lint
+        working-directory: feg/radius/lib/go/
+        run: |
+          for dir in */; do (cd "$dir" && $MAGMA_ROOT/feg/radius/src/run.sh lint); done
```

Comment:
looks strange that a loop is needed here - but I checked the docu and did not find another way to do this.